packages/ember-data/tests/unit/transform/number_test.js

```diff
@@ -20,7 +20,7 @@ test("#deserialize", function() {
   equal(transform.deserialize(undefined),       null);
   equal(transform.deserialize("1.1"),           1.1);
   equal(transform.deserialize(1.1),             1.1);
-  equal(transform.deserialize(new Number(1.1)), 1.1);
+  equal(transform.deserialize(Number(1.1)), 1.1);
```

Comment:
These ones we want to stay the same as they are testing Number objects (not primitives) are being deserialized correctly.